src/pybind/mgr/telemetry/module.py

```diff
@@ -1027,6 +1062,12 @@ def compile_report(self, channels: Optional[List[str]] = None) -> Dict[str, Any]
                 'total_bytes': df['stats']['total_bytes'],
                 'total_avail_bytes': df['stats']['total_avail_bytes']
             }
+            # basic_pool_usage collection (2/2)
```

Comment:
I think this is a different collection (basic_usage_by_class?) from the pools new addition, so we can create a new one for more clarity; what do you think?

Reply:
Sure, since it doesn't concern pools.